### PR TITLE
wip - added demo website installer

### PR DESCRIPTION
**Pull request by mickaelandrieu**
* Introduce a new ``demo:load-website`` command to create the demonstration website into backbee standard.

* Introduce a Fixtures loader that I plan to use in the BackBee installer to provide a new ``install a demonstration website ?`` option in the Web installer

* Move all assets from backbee-standard to DemoBundle repository => if the previous option is not checked, backbee standard stay"data free"

demo-import

### CODE REVIEW

File: Command/LoadDemoWebsiteCommand.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+ * Load demonstration website command.
+ *
+ * @author Mickaël Andrieu <mickael.andrieu@lp-digital.fr>
+ */
+class LoadDemoWebsiteCommand extends AbstractCommand
+{
+    /**
+     * @var BackBee\Bundle\DemoBundle\Fixtures\DemoWebsiteLoader
+     */
+    private $demoWebsiteLoader;
```
Comment (eric-chau) [inline Command/LoadDemoWebsiteCommand.php]: A shorter legit name could be simple ``$fixtureLoader`` as long as you are in the Demo bundle ;)

File: Command/LoadDemoWebsiteCommand.php [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            )
+        ;
+    }
+
+    /**
+     * @{inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->application = $this->getApplication()->getApplication();
```
Comment (eric-chau) [inline Command/LoadDemoWebsiteCommand.php]: hm? you wanted to write ``$this->getApplication()``?

File: Data/Fixtures/DemoWebsiteLoader.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    /**
+     * Update Website SQL entry according to the BackBee installation.
+     *
+     * @param     array    Website configuration, must contains 'domain' and 'label' keys
+     * @return    array    Returns domain and label set up in database
+     */
+    public function updateWebsite($configuration)
+    {
+        $domain = $configuration['domain'];
+        $label  = $configuration['label'];
```
Comment (eric-chau) [inline Data/Fixtures/DemoWebsiteLoader.php]: as long as this method is public you should do some checks about ``$configuration`` like if is an array and if ``domain`` and ``label`` keys exist.

Comment (mickaelandrieu) [inline Data/Fixtures/DemoWebsiteLoader.php]: this make sense, I will improve this behavior.

File: Data/Fixtures/DemoWebsiteLoader.php [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+     * @param string $filename
+     * @return void|RuntimeException
+     */
+    public function loadFixtures($filename)
+    {
+        $filePath = realpath($filename);
+
+        if (false === $filePath) {
+            $filePath = $filename;
+        }
```
Comment (eric-chau) [inline Data/Fixtures/DemoWebsiteLoader.php]: hm I don't get this test, what is it supposed to test?

**Comment by eric-chau** [comment]
Nice addition, good job @mickaelandrieu :+1:

File: Data/Fixtures/DemoWebsiteLoader.php (hunk 4 above)
Comment (mickaelandrieu) [inline Data/Fixtures/DemoWebsiteLoader.php]: ``realpath`` can return false, I want to be sure $filePath is a path

**Comment by mickaelandrieu** [comment]
Thank you for review @eric-chau

**Pull request merged**